Identity.BusinessLogic/Services/Implementations/EShopAuthenticationService.cs

```diff
@@ -0,0 +1,98 @@
+﻿using AutoMapper;
+using Identity.BusinessLogic.DTOs;
+using Identity.BusinessLogic.Services.Interfaces;
+using Identity.DataAccess.Entities.Constants;
+using Identity.DataAccess.Entities.Exceptions;
+using Identity.DataAccess.Models;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Logging;
+
+namespace Identity.BusinessLogic.Services.Implementations
+{
+    public class EShopAuthenticationService(UserManager<User> userManager, ITokenService tokenService, IMapper mapper, ILogger<EShopAuthenticationService> logger) : IAuthenticationService
```

Comment:
your interface is called IAuthenticationService, name the implementation of this service with the appropriate name